src/credentials.rs

```diff
@@ -200,6 +199,23 @@ impl ChannelCredentials {
     pub fn as_mut_ptr(&mut self) -> *mut GrpcChannelCredentials {
         self.creds
     }
+
+    /// Attempts to construct a `ChannelCredentials` that is authenticated with
+    /// Google OAuth credentials.
+    pub fn google_default_credentials() -> Result<ChannelCredentials, String> {
```

Comment:
Please use the error from `src/error.rs`  instead.

Reply:
@BusyJay Done - Let me know if there is anything else. Thanks